src/components/SignupForm/useFormFieldsConfig.tsx

```diff
@@ -0,0 +1,55 @@
+import SignUpFormDataInterface from './interfaces/SignupFormDataInterface';
+import FormFieldConfigInterface from './interfaces/FormFieldConfigInterface';
+
+const useFormFieldsConfig = (
+  formData: SignUpFormDataInterface,
+  errorMessageVisible: {
+    name: boolean;
+    email: boolean;
+    password: boolean;
+    repeatPassword: boolean;
+  }
+): FormFieldConfigInterface[] => {
+  const formFieldsConfig = [
+    {
+      key: 'input1',
+      label: 'Email',
+      type: 'email',
+      name: 'email',
+      isError: formData.email === '' && errorMessageVisible.email,
+      helperText: formData.email === '' && errorMessageVisible.email ? "Email can't be empty" : '',
```

Comment:
it errorMessage not helperText. Helper text could be in tooltip for example next to label